src/Components/CreateImageWizard/steps/Packages/PackageRecommendations.tsx

```diff
@@ -45,6 +45,18 @@ const PackageRecommendations = () => {
   const version = releaseToVersion(distribution);
   const undebouncedPackages = useAppSelector(selectPackages);
   const packages = useDebounce(undebouncedPackages);
+
+  const getDistributionForRecommendations = (distribution: string): string => {
+    const distroLower = distribution.toLowerCase();
+    if (distroLower.includes('rhel') && distroLower.includes('8'))
+      return 'rhel-8';
+    if (distroLower.includes('rhel') && distroLower.includes('9'))
+      return 'rhel-9';
+    if (distroLower.includes('rhel') && distroLower.includes('10'))
+      return 'rhel-10';
+    if (distroLower.includes('rhel')) return 'rhel-9';
+    return 'rhel-9';
+  };
```

Comment:
Distribution is already in the needed format, isn't it?

![image](https://github.com/user-attachments/assets/f6c66a1b-8790-471a-aba5-31364ed4f669)